### DISCUSSION TITLE
Performance for constraint versus  unconstraint plan type

### DISCUSSION THREAD

**Discussion by gibhsu** [frePPLe/frepple#478] [Q&A]
If currently I don't set any resource constraints and frozen period. And if I allow manufacturing plan to be in the past.
Will the demand versus WIP generates the same results when I switch to plan type from constraint to unconstraint.
The reason I ask is because when I tested the same data set, the processing time has a big gate for 12 hours different.

**Comment by jdetaeye** [comment]
>> Will the demand versus WIP generates the same results when I switch to plan type from constraint to unconstraint.

The material and capacity allocation will be very different in a constrained plan. 

>>The reason I ask is because when I tested the same data set, the processing time has a big gate for 12 hours different.

That's not the right argument to select the plan type you need.  You need to diagnose the root cause of the slow search in the log file of the constrained plan.
